sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java

```diff
@@ -141,6 +143,34 @@ public CaseDetailedTableViewSteps(
               "Reporting user is not correct");
           softly.assertAll();
         });
+
+    When(
+        "I back to Case Directory using case list button",
```

Comment:
refactor a bit this step and you can also add a wait for a specific element to be present, so we can avoid flaky situations or forcing us to add a wait in all other methods which can be called after this one